src/edu/first/module/sensors/GyroscopeModule.java

```diff
@@ -11,7 +11,7 @@
  * @since May 22 13
  * @author Joel Gallant
  */
-public class GyroscopeModule extends Module.StandardModule implements Gyroscope {
+public class GyroscopeModule extends Module.StandardModule implements Gyroscope, Gyro {
```

Comment:
Incorrect, should change line 16 to AnalogGyroscope

As is, this recursively constructs, and basically does nothing